samples/subsys/zbus/msg_subscriber/README.rst

```diff
@@ -0,0 +1,77 @@
+.. _zbus-msg-subscriber-sample:
+
+Message subscriber
+##################
```

Comment:
Would be nice to use the new `zephyr:code-sample` directive ;) See other samples to adopt a similar style/tone

Reply:
I forgot that. Thank you for remembering it.